tests/pipelines/test_pipelines_common.py

```diff
@@ -179,6 +179,18 @@ def is_test_to_skip(test_casse_name, config_class, model_architecture, tokenizer
             #       fails this test case. Skip for now - a fix for this along with the initial changes in PR #20426 is
             #       too much. Let `ydshieh` to fix it ASAP once #20426 is merged.
             to_skip = True
+        elif config_class.__name__ == "LayoutLMv2Config" and test_casse_name in [
+            "QAPipelineTests",
+            "TextClassificationPipelineTests",
+            "TokenClassificationPipelineTests",
+            "ZeroShotClassificationPipelineTests",
+        ]:
+            # `LayoutLMv2Config` was never used in pipeline tests (`test_pt_LayoutLMv2Config_XXX`) due to lack of tiny
+            # config. With new tiny model creation, it is available, but we need to fix the failed tests.
+            to_skip = True
+        elif test_casse_name == "DocumentQuestionAnsweringPipelineTests" and not tokenizer_name.endswith("Fast"):
+            # This pipeline uses `sequence_ids()` which is only available for fast tokenizers.
+            to_skip = True
```

Comment:
We should not test LayoutLMv2 at all then, until we have the common tester for pipelines and we can overwrite the common tests in individual test files. The common test should not contain a list of tests on configs/test names.

Reply:
Agree! This big `is_test_to_skip`  is added in #20426 to server as a record + to pass CI in order to merge.

Reply:
(BTW, `LayoutLMv2`  was never used in pipeline testing before #20426, but after that PR, it works in `DocumentQuestionAnsweringPipelineTests`)

Reply:
I agree with @sgugger I think,

Let's escape individually `layoutlmv2` in all the pipeline it's attempted to be called.
IT makes it more obvious to readers that the model is broken, and we can enable them one by one.

Reply:
I am going to start implementing common pipeline testing (as common model testing), and the list here is not only to skip tests, but more importantly to record what tests need to be fixed.

As it is just temporarily, and `is_test_to_skip` already contains lot of skip cases, I don't think it hurts (more) to add one more case (and again, at least I will know clearly what tests should be fixed or to be overwritten in the common pipeline testings PR)